diff -up range-v3-0.9.1/include/range/v3/all.hpp.omv~ range-v3-0.9.1/include/range/v3/all.hpp
--- range-v3-0.9.1/include/range/v3/all.hpp.omv~	2019-09-15 19:08:37.306735210 +0200
+++ range-v3-0.9.1/include/range/v3/all.hpp	2019-09-15 19:08:45.157510920 +0200
@@ -19,6 +19,7 @@
 #include <range/v3/core.hpp>
 #include <range/v3/functional.hpp>
 #include <range/v3/iterator.hpp>
+#include <range/v3/iterator_range.hpp>
 #include <range/v3/numeric.hpp>
 #include <range/v3/range.hpp>
 #include <range/v3/utility.hpp>
